operator/config/manifests/bases/rhacs-operator.clusterserviceversion.yaml

```diff
@@ -9,6 +9,13 @@ metadata:
     createdAt: "1999-12-31T23:59:59Z"
     description: Red Hat Advanced Cluster Security (RHACS) operator provisions the
       services necessary to secure each of your OpenShift and Kubernetes clusters.
+    features.operators.openshift.io/disconnected: "true"
+    features.operators.openshift.io/fips-compliant: "false"
+    features.operators.openshift.io/proxy-aware: "true"
+    features.operators.openshift.io/tls-profiles: "false"
```

Comment:
@stackrox/draco Could you please confirm whether we can set `features.operators.openshift.io/tls-profiles` to `true`?

Reply:
I'm not aware of any option to set ciphers for the operator or its operands. So I think the current setting in this PR `features.operators.openshift.io/tls-profiles: "false"` is correct.